ckanext/datagovuk/helpers.py

```diff
@@ -297,3 +297,8 @@ def roles():
 
 def google_analytics_tracking_id():
     return config.get('ckan.google_analytics_tracking_id')
+
+def is_central_gov_organogram(pkg_name):
+    pkg_dict = Package.by_name(pkg_name).as_dict()
+    pkg_extras = pkg_dict.get('extras')
```

Comment:
I think we should use `.get('extras', {})` here so we get an empty dictionary if the key doesn't exist, otherwise in the next line we would get a `TypeError`. Or perhaps if we expect the key to exist should be just do `pkg_dict['extras']`?

Reply:
I'll also update the other bits of code